FLEDGE.md

```diff
@@ -784,7 +810,26 @@ When `joinCount` is passed to `generateBid()`, no noising or bucketing is applie
 
 These signals were requested in [issue 435](https://github.com/WICG/turtledove/issues/435). The signals are intented to ship in Chrome M114, they will no longer be available for event level reporting when event level reporting is retired.
 
-#### 5.3 Losing Bidder Reporting
+#### 5.3 Reporting in Multi-Currency Auctions
+
+In auctions that involve multiple currencies, there may be values with different units floating around, which makes aggregated information incomprehensible, and event-level information hard to process, potentially requiring participants to interpret currencies of jurisdictions they do no business in.
+
+To help deal with this scenario, an optional mode is available that converts all bid-related information to seller's preferred currency (in component auctions, reporting for it is for that component's seller). This is configured via the `sellerCurrency` setting in each auction configuration.
+
+Regardless of configuration, the bid passed to `reportWin()` and `reportResult()` is always the bid that participated in that particular level of the auction (which, for top-level auction `reportResult()`, may be the bid altered by the component auction), subject to the usual rounding rules. Since passing the returned currency tags around could leak an unacceptable amount of information, the values provided for `browserSignals.bidCurrency` to those methods are the currencies required by the auction configuration (i.e. `sellerCurrency` and `perBuyerCurrencies`), or `'???'` if unspecified.
+
+(Warning: `reportResult` did not follow this rule in Chrome earlier than M116)
+
+All other values, including `highestScoringOtherBid` and winning bids provided to private aggregation and debug event-level loss reporting depend on the `sellerCurrency` setting.
+
+If `sellerCurrency` is not set, all bids in those other ("mixed currency") contexts are passed as-is, and all the corresponding currency tags (`bidCurrency`, `highestScoringOtherBidCurrency`) are redacted to `'???'`.
```

Comment:
So to verify, if the auction config's `perBuyerCurrency` for a buyer and `generateBid()`'s `bidCurrency` for a buyer are both `USD`, but `sellerCurrency` is not set, then `browserSignals.bidCurrency` is `???`?

Reply:
Yes

Reply:
In that case, can we clarify or rephrase the term "mixed currency" here?  It feels like it could be homogeneous currencies but underspecified, for example if `perBuyerCurrency` and `bidCurrency` match but `sellerCurrency` is not specified.

Reply:
This is supposed to define it as not being the cases listed on line 821....

Reply:
This needs to be made clearer.  Perhaps use the term "matching currencies" on line 821 and very clearly define the conditions when this term/situation applies, and then use the term "non-matching currencies" here and and very clearly define that this applies whenever the conditions for "matching currencies" are not met.

Reply:
That's not the term I am using; but at any rate, rephrased some.